osu.Framework/Graphics/Containers/SearchContainer.cs

```diff
@@ -94,26 +113,26 @@ protected override void Update()
         private void performFilter()
         {
             string[] terms = (searchTerm ?? string.Empty).Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Small suggestion while you are already working on this code:
```suggestion
            string[] terms = (searchTerm ?? string.Empty).Split(' ', StringSplitOptions.RemoveEmptyEntries);
```
To avoid the allocation.